pkg/mtbroker/common_test.go

```diff
@@ -0,0 +1,57 @@
+/*
```

Comment:
Is this test really necessary? I don't see much added value for it.

Reply:
I agree, my goal is to make Prow's coverage check happy. If we get rid of it coverage check will fail every time we update this file. What do you think?